front/src/modules/auth/components/RequireNotAuth.tsx

```diff
@@ -34,15 +36,32 @@ export function RequireNotAuth({
 }): JSX.Element {
   const navigate = useNavigate();
 
+  const [currentUser] = useRecoilState(currentUserState);
   const isLogged = useIsLogged();
 
   useEffect(() => {
-    if (isLogged) {
+    if (
```

Comment:
Sounds like this component is not only testing whether the user is logged in but also if they completed the onboarding. What about renaming from `RequireNotAuth` to `RequireOnboardingComplete` ?